react/package.json

```diff
@@ -1,17 +1,17 @@
 {
-  "name": "vtex.header",
-  "description": "Customizable header",
   "dependencies": {
-    "@types/graphql": "^14.2.0",
-    "@types/jest": "^24.0.11",
-    "@types/node": "^11.13.2",
-    "@types/ramda": "^0.26.8",
-    "@types/react": "^16.8.13",
-    "@types/react-intl": "^2.3.17",
     "@vtex/css-handles": "^1.1.3",
     "classnames": "^2.2.6",
```

Comment:
I think we don't even need `classnames` and `react-intl`

Reply:
we do need intl. classnames tho, good catch

Reply:
(`defineMessages` is used a bunch of times)